clients/java/zms/pom.xml

```diff
@@ -141,6 +141,16 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/ZMSRDLGeneratedClient.*</exclude>
```

Comment:
I noticed that the exclude list only works in the main pom.xml where code.coverage.min is referenced. So we should exclude ZMSRDLGeneratedClient and any other classes we want to exclude in yahoo/athenz/pom.xml (the parent pom.xml for all the other projects).